src/components/assets/icons/SSIEntityIcon/index.tsx

```diff
@@ -1,29 +1,41 @@
 import React, { FC } from 'react'
-import { ColorValue, View } from 'react-native'
-import Svg, { Circle, Path } from 'react-native-svg'
+import { ColorValue } from 'react-native'
+import { connect } from 'react-redux'
 
 import { entities, fonts } from '../../../../styles/colors'
+import {
+  SSIEntityIconStyled as Avatar,
+  SSITextH2LightStyled as AvatarText
+} from '../../../../styles/components'
+import { RootState } from '../../../../types'
+import { getDefaultUserNameInitials } from '../../../../utils/UserUtils'
 
 export interface IProps {
   size?: number
-  color?: ColorValue
-  backgroundColor?: ColorValue
+  fontColor?: ColorValue
+  bgColor?: ColorValue
+  initialsOfUserName: string
 }
 
 const SSIEntityIcon: FC<IProps> = (props: IProps): JSX.Element => {
-  const { size = 34, color = fonts.light, backgroundColor = entities['100'] } = props
+  const { size = 34, fontColor = fonts.light, bgColor = entities['100'], initialsOfUserName = '?' } = props
```

Comment:
lets rename initialsOfUserName  to initials, as there are no other initials

Reply:
The variable was removed.